ibis/expr/api.py

```diff
@@ -2094,8 +2094,8 @@ def range(start, stop, step) -> ir.ArrayValue:
     Integer ranges are supported, as well as timestamp ranges.
 
     ::: {.callout-note}
-    `start` is inclucive and `stop` is exclusive, just like Python's builtin
-    [`range`](range).
+    `start` is inclusive and `stop` is exclusive, just like Python's builtin
+    [`range`](:py:class:`range`).
```

Comment:
```suggestion
    [](`range`).
```

I believe this should fix the reference error.

Reply:
Ah, didn't see your comment in the top-level PR.

I think we should probably avoid the Sphinx syntax since it kind of gives the impression that people should write Sphinx.

Admittedly, it's probably unlikely that someone will come across this and then additionally conclude that Sphinx is in use, but if there's a way we can do it that matches everything else like it (referencing a builtin object) then I think we should.

Reply:
I swear I tried the suggested code before and it didn't render, but I went back to confirm and it works.

Interlinks on quartodoc do support the sphinx syntax, just in case we ever need it for specifying an ambiguous link, but I'll switch this one to use the simpler syntax.

Reply:
> Interlinks on quartodoc do support the sphinx syntax

😮